### PR TITLE
i18n(es): Updated `vercel.mdx`

### PR DESCRIPTION
**Pull request by dreyfus92**
#### What kind of changes does this PR include?

- New or updated content
- Changes to the docs site code

#### Description

- What does this PR change? Give us a brief description.
Updated `vercel.mdx` based on this commit c0ef26d2dacc3b7cebca4cad6e88bae393048feb

### CODE REVIEW

File: src/content/docs/es/guides/integrations-guide/vercel.mdx [hunk 1]
```diff
@@ -264,6 +264,77 @@ Puedes usar el middleware de Vercel para interceptar una solicitud y redirigirla
...
+*   debe **regresar** un `object`;
+*   acepta un objeto con una `request` y `context` como propiedades;
+*   `request` es tipada como [`Request`](https://developer.mozilla.org/es/docs/Web/API/Request);
+*   `context` es tipado como [`RequestContext`](https://vercel.com/docs/concepts/functions/edge-functions/vercel-edge-package#requestcontext);
+
+#### Limitaciones y restricciones
+
+Cuando optes por esta característica, hay algunas restricciones a tener en cuenta:
+
+*   El middleware de Vercel Edge sera siempre la **primera** función en recibir la `Request` y la última función en recibir la `Response`. Esta es una restricción arquitectónica que sigue los [límites establecidos por Vercel](https://vercel.com/docs/concepts/functions/edge-middleware).
```
Comment (Waxer59) [inline src/content/docs/es/guides/integrations-guide/vercel.mdx:334]:
```suggestion
*   El middleware de Vercel Edge será siempre la **primera** función en recibir la `Request` y la última función en recibir la `Response`. Esta es una restricción arquitectónica que sigue los [límites establecidos por Vercel](https://vercel.com/docs/concepts/functions/edge-middleware).
```

File: src/content/docs/es/guides/integrations-guide/vercel.mdx [hunk 2]
```diff
@@ -237,7 +237,7 @@ export default defineConfig({
 
 ### Middleware de Vercel
```
Comment (Waxer59) [inline src/content/docs/es/guides/integrations-guide/vercel.mdx:238]:
```suggestion
### Middleware de Vercel Edge
```

File: src/content/docs/es/guides/integrations-guide/vercel.mdx [hunk 3]
```diff
@@ -264,6 +264,77 @@ Puedes usar el middleware de Vercel para interceptar una solicitud y redirigirla
 **¿Intentando reescribir?** Actualmente, reescribir una solicitud con middleware solo funciona para archivos estáticos.
 :::
 
+### Middleware de Vercel Edge con middleware de Astro
+
+El adaptador `@astrojs/vercel/serverless` puede crear automáticamente el middleware de Vercel Edge a partir de un middleware de Astro en tu base de código.
```
Comment (Waxer59) [inline src/content/docs/es/guides/integrations-guide/vercel.mdx:269]:
```suggestion
El adaptador `@astrojs/vercel/serverless` puede crear automáticamente el middleware de Vercel Edge a partir de un middleware de Astro en tu código base.
```

File: src/content/docs/es/guides/integrations-guide/vercel.mdx [hunk 4]
```diff
@@ -237,7 +237,7 @@ export default defineConfig({
 
 ### Middleware de Vercel
 
-Puedes usar el middleware de Vercel para interceptar una solicitud y redirigirla antes de enviar una respuesta. El middleware de Vercel puede ejecutarse en implementaciones de Edge, SSR y estáticas. No necesitas instalar `@vercel/edge` para escribir middleware, pero sí necesitas instalarlo para usar características como la geolocalización. Para obtener más información, consulta la [documentación de middleware de Vercel](https://vercel.com/docs/concepts/functions/edge-middleware).
+Puedes usar el middleware de Vercel para interceptar una solicitud y redirigirla antes de enviar una respuesta. El middleware de Vercel puede ejecutarse en implementaciones de Edge, SSR y estáticas. Es posible que no necesites instalar este paquete para tu middleware. `@vercel/edge` es solo necesario para usar algunas usar algunas características de middleware como la geolocalización. Para obtener más información, consulta la [documentación de middleware de Vercel](https://vercel.com/docs/concepts/functions/edge-middleware).
```
Comment (Waxer59) [inline src/content/docs/es/guides/integrations-guide/vercel.mdx:240]:
```suggestion
Puedes usar el middleware de Vercel Edge para interceptar una solicitud y redirigirla antes de enviar una respuesta. El middleware de Vercel puede ejecutarse en implementaciones de Edge, SSR y estáticas. Es posible que no necesites instalar este paquete para tu middleware. `@vercel/edge` es solo necesario para usar algunas usar algunas características de middleware como la geolocalización. Para obtener más información, consulta la [documentación de middleware de Vercel](https://vercel.com/docs/concepts/functions/edge-middleware).
```

**Review by Waxer59 — COMMENTED** [review]
Buena PR 🤠, te dejo unas sugerencias

**Comment by dreyfus92** [comment]
Muchas gracias por la revisión @Waxer59 😁

**Review by Waxer59 — APPROVED** [review]
LGTM ✔

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Good job peeps 🙌

**Pull request merged**